pkg/cri/sbserver/instrumented_service.go

```diff
@@ -1003,6 +1013,8 @@ func (in *instrumentedAlphaService) ListImages(ctx context.Context, r *runtime_a
 	if err := in.checkInitialized(); err != nil {
 		return nil, err
 	}
+	ctx, span := tracing.StartSpan(ctx, makeSpanName("ListImages"))
+	defer tracing.StopSpan(span)
```

Comment:
`defer span.Close()` or `defer span.Stop()` feels more natural.

Reply:
yeah, that is a good point. I can make the change and wrap `trace.Span` object.  As for context propagation, I am thinking I will  keep the existing logic. I can modify `StartSpan` and `CurrentSpan` to return me the wrapper Span, but keep span context propagation as is.

Reply:
I'd prefer a wrapper object, but I don't want to block this PR. That's fine to keep it as is and/or follow up later.

Reply:
Sounds good to me. I will send a follow up PR adding the wrapper object and restructure some of the current span methods.